### PR TITLE
replace NodePort with ClusterIP

### PR DESCRIPTION
**Pull request by dati18**
Replace NodePort with ClusterIP

https://phabricator.wikimedia.org/T295119

### CODE REVIEW

**Review by deer-wmde — CHANGES_REQUESTED** [review]
I tested it in my local cluster and it seems to work! :partying_face: 

But I noticed there are still some charts with NodePort:
- tool-widar
- tool-quickstatements
- tool-cradle
- ui

**Comment by dati18** [comment]
> I tested it in my local cluster and it seems to work! partying_face
> 
> But I noticed there are still some charts with NodePort:
> 
> * tool-widar
> * tool-quickstatements
> * tool-cradle
> * ui

sure, lemme check that...

**Comment by dati18** [comment]
> I tested it in my local cluster and it seems to work! partying_face
> 
> But I noticed there are still some charts with NodePort:
> 
> * tool-widar
> * tool-quickstatements
> * tool-cradle
> * ui

I remember I didn't change them because there are a few errors. I have fixed most of it...

**Review by deer-wmde — APPROVED** [review]

File: charts/api/README.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 ## Changelog
 
+- 0.10.1: Change service from `NodePort` to `ClusterIP`
```
Comment (addshore) [inline charts/api/README.md:5]:
```suggestion
- 0.10.2: Change service from `NodePort` to `ClusterIP`
```

**Review by addshore — CHANGES_REQUESTED** [review]

**Review by addshore — APPROVED** [review]

**Pull request merged**